server/src/main/java/org/apache/druid/client/HttpServerInventoryView.java

```diff
@@ -540,7 +540,7 @@ private class DruidServerHolder
             smileMapper,
             httpClient,
             inventorySyncExecutor,
-            new URL(druidServer.getScheme(), hostAndPort.getHostText(), hostAndPort.getPort(), "/"),
+            new URL(druidServer.getScheme(), hostAndPort.getHost(), hostAndPort.getPort(), "/"),
```

Comment:
why is this change needed?

Reply:
@abhishekagarwal87 it is a guava api change. I had to do the same for compatibility in my companies fork when I upgraded guava. should be safe